src/smtp/SMTPServer.ts

```diff
@@ -89,6 +89,16 @@ export default class SMTPServer {
 				const email = msg.split(":")[1].split(">")[0].replace("<", "")
 				const [username, domain] = email.split("@")
 
+				if (info.from.endsWith(`@${getConfig("host")}`)) {
+					// This is an outgoing mail, we need to forward it to another server
```

Comment:
Question: shouldn't you push email to info.to regardless of what info.from is?

Reply:
this is just a check to see if the mail is coming from one of our users.